app/admin/dashboard.rb

```diff
@@ -11,43 +11,74 @@
           attributes_table_for Event.includes(:sessions, :rooms, :timeslots).current_event do
             row :name
             row :date
-            row "Show Schedule" do
-              settings.show_schedule
-            end
             row "Allow New Sessions" do
               settings.allow_new_sessions
             end
+            row "Show Schedule" do
+              settings.show_schedule
+            end
             row "# of Sessions" do |event|
-              event.sessions.size
+              event.sessions_count
             end
             row "# of Rooms" do |event|
-              event.rooms.size
+              event.rooms_count
             end
             row "# of Timeslots" do |event|
-              event.timeslots.size
+              event.timeslots_count
             end
           end
         end
       end
     end
 
     panel "Current Event Sessions" do
-      table_for Event.includes(:sessions).current_event.sessions.includes(:presenters, :attendances, :timeslot, :room).order(:timeslot_id) do
-        column :title do |session|
-          link_to session.title, admin_session_path(session)
+      # Define allowed sort columns and their database equivalents
```

Comment:
This whole chunk of code is duplicated on both `dashboard.rb` and`events.rb`, but duplicating the code was easier than trying to figure out how to DRY it up by adding reusable helper for use within active admin 🤷